src/ui/public/notify/notify.less

```diff
@@ -8,21 +8,21 @@
     margin: 0;
     padding: 0;
     list-style: none;
+  }
+
+  .toaster-countdown {
+    .badge {
```

Comment:
Instead of nesting classes, we can create a [descendant class](https://github.com/elastic/kibana/blob/master/style_guides/css_style_guide.md#create-descendant-classes-to-represent-child-components-which-cant-stand-on-their-own) here:

```scss
.toaster-countdown__badge {
  color: @white;
}

.toaster-countdown {
  &:hover {
    .toaster-coutdown__badge {
      font-size: 0;
      &:after {
        font-size: @font-size-small;
        content: attr(hover-text);
      }
    }
}
```